src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ProjectTypeRegistration.cs

```diff
@@ -16,7 +16,8 @@
     DisableAsynchronousProjectTreeLoad = true,
     PossibleProjectExtensions = "vbproj",
     NewProjectRequireNewFolderVsTemplate = true,
-    SupportsCodespaces = true)]
+    SupportsCodespaces = true,
+    SupportsReloadFreeBranchSwitch = true)]
```

Comment:
As mentioned on the CPS PR, could the property name be `SupportsSolutionChangeWithoutReload`?

```suggestion
    SupportsSolutionChangeWithoutReload = true)]
```

Reply:
Which is a very good point, because you'd do this regardless of whether it was a branch switch or not.